### PR TITLE
Unify order of tokens. fix #42

### LINKED ISSUE #42: The currencies in the “Add Asset” screen are ordered differently from wallet page
The currencies in the “Add Asset” screen are ordered differently from the screen when you set the wallet for the first time where Testnet and Mainnet currencies are separated. I personally prefer when they are separated.

### PR DESCRIPTION
**Pull request by imstar15**
**List**

截屏2020-04-29 22 00 53

**Add Tokens**

截屏2020-04-29 18 30 26

**Key Setting**

截屏2020-04-29 18 30 46

### CODE REVIEW

File: src/common/common.js [hunk 1]
```diff
@@ -403,6 +403,24 @@ const common = {
...
+    return tokens.sort((a, b) => {
+      if (a.type !== b.type) {
+        return b.type === 'Testnet' ? -1 : 1;
+      }
+      let symbolIndexA = _.findIndex(supportedTokens, (token) => a.symbol === token);
+      // If token is not found in supportedTokens, indicating it is a custom token, Should be at the end of the list
+      symbolIndexA = symbolIndexA !== -1 ? symbolIndexA : Number.MAX_SAFE_INTEGER;
+      let symbolIndexB = _.findIndex(supportedTokens, (token) => b.symbol === token);
+      symbolIndexB = symbolIndexB !== -1 ? symbolIndexB : Number.MAX_SAFE_INTEGER;
+      return symbolIndexA - symbolIndexB;
```
Comment (chrisli30) [inline src/common/common.js:421]: What if you get indexA === -1 and indexB === -1?

If the return is 0, does that mean sometimes you get A, B, and other times get B, A?

Comment (chrisli30) [reply src/common/common.js:421]: The best solution is to sort they by time added to the list.

or you can sort by symbol name.

Comment (imstar15) [reply src/common/common.js:421]: Good question!

Comment (imstar15) [reply src/common/common.js:427]: Fixed

**Pull request merged**